src/test/.eslintrc.js

```diff
@@ -3,7 +3,7 @@ module.exports = {
   env: {
     jest: true,
   },
-  plugins: ['testing-library'],
+  plugins: ['jest', 'testing-library'],
```

Comment:
Not sure what this is for, but this is part of the normal configuration, so I added it too.